x/rewards/client/cli/query_provider_reward.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/lavanet/lava/x/rewards/types"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdQueryProviderReward() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "provider-reward [provider] {chain-id}",
+		Short: "Query the total rewards for a given provider and chain",
+		Long: `args: 
+	[provider] Provider address for which we want to query the rewards
+	{chain-id} Chain for which we want to view the rewards of the provider
+		`,
+		Example: "lavad query rewards provider-reward <provider's address> ETH1",
```

Comment:
put another example of: "lavad query rewards provider-reward <provider's address>" to emphasize that chain-id is optional